frontend/tests/integration/work-package-details-relations-spec.js

```diff
@@ -0,0 +1,51 @@
+var chai = require('chai');
```

Comment:
@floriank Merging #2617 means Hound no longer picks up the right `.jshintrc` settings... We should adjust the `config_file` path in `.hound.yml`.